app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -152,6 +155,18 @@ fun HomeScreen(
 
     if (uiState.tab == Tab.BROWSE) {
```

Comment:
imp: I think we could reset the searchBar state when we switch to the Upcoming tab so that after switching tabs, the text in search bar is also reset :)

Reply:
Done!